pandas/tests/io/excel/test_base.py

```diff
@@ -80,46 +64,11 @@ def get_csv_refdf(self, basename):
         dfref = read_csv(pref, index_col=0, parse_dates=True, engine='python')
         return dfref
 
-    def get_excelfile(self, basename, ext):
-        """
-        Return test data ExcelFile instance.
-
-        Parameters
-        ----------
-
-        basename : str
-            File base name, excluding file extension.
-
-        Returns
-        -------
-
-        excel : io.excel.ExcelFile
-        """
-        return ExcelFile(os.path.join(self.dirpath, basename + ext))
-
-    def get_exceldf(self, basename, ext, *args, **kwds):
-        """
-        Return test data DataFrame.
-
-        Parameters
-        ----------
-
-        basename : str
-            File base name, excluding file extension.
-
-        Returns
-        -------
-
-        df : DataFrame
-        """
-        pth = os.path.join(self.dirpath, basename + ext)
-        return read_excel(pth, *args, **kwds)
-
 
 class ReadingTestsBase(SharedItems):
     # This is based on ExcelWriterBase
 
-    @pytest.fixture(autouse=True, params=['xlrd', None])
+    @pytest.fixture(autouse=True, params=[None])
```

Comment:
maybe a better name for `test_base` would be `test_common` or `test_all_engines`. this split is creating too much code duplication. maybe a split on reader/writer tests first before splitting out xlrd